extensions/wikia/AdEngine/js/PageFairDetection.js

```diff
@@ -0,0 +1,71 @@
+define('ext.wikia.adEngine.pageFairDetection', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.utils.scriptLoader',
+	'wikia.browserDetect',
+	'wikia.document',
+	'wikia.log',
+	'wikia.window'
+], function (adContext, scriptLoader, browserDetect, doc, log, win) {
+	'use strict';
+
+	var context = null,
+		logGroup = 'ext.wikia.adEngine.pageFairDetection';
+
+	function getContext() {
+		if (context === null) {
+			context = adContext.getContext();
+		}
+		return context;
+	}
+
+	function skinIsMobile() {
```

Comment:
Can we inline this function?
(it's used once and I am not sure that it helps with readability)